esmvaltool/cmorizers/data/formatters/datasets/jra_55.py

```diff
@@ -0,0 +1,177 @@
+                        standard_name='time',
+                        long_name='time',
+                        units=time_units)
+
+                    tmp_cubes[i].add_aux_coord(time_coord)
+                    tmp_cubes[i].remove_coord('originating_centre')
+
+                    i = i + 1
+
+            # Some JRA-55 GRIB files contain two fields: "surface" and
+            # "top of the atmosphere". As
+            # message.sections[1]['indicatorOfTypeOfLevel'] always gives
+            # 'sfc', a distinction between "surface" and "top of the
+            # atmosphere" is not possible (bug?).
+            # If "start_element" is given in the JRA-55 CMOR config file
+            # (esmvaltool/cmorizers/data/cmor_config/JRA-55.yml), we simply
+            # extract every second cube from the list of cube starting
+            # with field "start_element" as the fields "surface" and "top of
+            # the atmosphere" are alternating in the GRIB file.
+            # If "start_element" is not specified in the JRA-55 CMOR config
+            # file, no selection of cubes is done before merging into a
+            # single cube.
+
+            if start_element is not None:
+                cubelist.append(tmp_cubes[start_element::2].merge_cube())
+            else:
+                cubelist.append(tmp_cubes.merge_cube())
+        else:
+            tmp_cubes[0].remove_coord('originating_centre')
+            cubelist = tmp_cubes
+
+    return cubelist
+
+
+def _extract_variable(short_name, var, in_files, cfg, out_dir):
+    """Extract variable."""
+    # load data (returns a list of cubes)
+
+    cubes = _load_jra55_grib(in_files, var)
+
+    # apply operators (if any)
+
+    if len(cubes) > 1:
+        if var.get('operator', '') == 'sum':
+            # Multiple variables case using sum operation
+            cube = None
+            for in_cube in cubes:
+                if cube is None:
+                    cube = in_cube
+                else:
+                    cube += in_cube
+        elif var.get('operator', '') == 'diff':
+            # two variables case using diff operation
+            if len(cubes) != 2:
+                errmsg = (f'operator diff selected for variable {short_name} '
+                          f'expects exactly two input variables and two input '
+                          f'files')
+                raise ValueError(errmsg)
+            cube = cubes[0]
+            cube -= cubes[1]
+        else:
+            oper = var.get('operator')
+            raise ValueError(
+                f'multiple input files found for variable {short_name} '
+                f'with unknown operator {oper}')
+    else:
+        cube = cubes[0]
+
+    # Fix metadata
+    cmor_info = cfg['cmor_table'].get_variable(var['mip'], short_name)
+    attrs = copy.deepcopy(cfg['attributes'])
+    attrs['mip'] = var['mip']
+    utils.fix_var_metadata(cube, cmor_info)
+
+    # fix z-coordinate (if present)
+
```

Comment:
Thanks for the other changes @axel-lauer. Not sure if this one got missed or you want to keep the blank line for a reason? It would now be line 135 that this refers to.